team_red/models/model.py

```diff
@@ -21,7 +21,9 @@ def model_post_init(self, __context: Any) -> None:  # noqa: ANN401
     @computed_field  # type: ignore[misc]
     @property
     def parameters(self) -> List[str]:
-        return [fn for _, fn, _, _ in Formatter().parse(self.text) if fn is not None]
+        dup = {fn for _, fn, _, _ in Formatter().parse(self.text) if fn is not None}
+        uniq = set(dup)
+        return sorted(uniq)
```

Comment:
`{f for f in foo}` is a set already

Reply:
It definitely is more readable but it is unnecessary when `x = {fn for _, fn ...}` is sufficient. Ruff seems to like the latter.